kernel/classes/datatypes/ezuser/ezuser.php

```diff
@@ -27,6 +27,8 @@ class eZUser extends eZPersistentObject
     const PASSWORD_HASH_MYSQL = 4;
     /// Passwords in plaintext, should not be used for real sites
     const PASSWORD_HASH_PLAINTEXT = 5;
+    /// Passwords in bcrypt of password
+    const PASSWORD_BCRYPT = 6;
```

Comment:
@michalpipa since password_hash is only present on PHP 5.5, I'm currently using https://github.com/ircmaxell/password_compat has you can see in the composer.json